src/main/java/mate.academy/service/ProductParser.java

```diff
@@ -1,7 +1,9 @@
 package mate.academy.service;
 
+import mate.academy.lib.Component;
 import mate.academy.model.Product;
 
+@Component
```

Comment:
we dont need this annotation here, it should be over class-implementation of this interface